pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/summaryBar.tsx

```diff
@@ -72,6 +72,7 @@ export const ClusterNodeTotals: React.FC = () => {
         </span>
       }
       value={nodeCounts.total}
+      numberAlert={false}
```

Comment:
I checked in react boolean does not come with a default value, I think it is safer to set it as false?